pkg/eval/builtin_fn_num.go

```diff
@@ -588,9 +587,12 @@ func rem(a, b int) (int, error) {
 // ▶ 6
 // ```
 
+var ErrRandIntArgs = errors.New("$low >= $high")
+
 func randint(low, high int) (int, error) {
 	if low >= high {
-		return 0, ErrArgs
+		return 0, errs.BadValue{What: "low value",
```

Comment:
Since the high value comes after the low value, it's more intuitive to think of the constraint that `high` should be larger than `low` and invert the logic of the error here.